packages/modules/itwin-viewer-react/src/components/iModel/IModelViewer.tsx

```diff
@@ -42,13 +44,10 @@ export const IModelViewer: React.FC<ModelProps> = ({
     }
   }, [frontstages]);
 
-  const frameworkVersion =
-    (process.env.REACT_APP_FRAMEWORK_VERSION as FrameworkVersion) || "1";
-
   // there will always be at least one (for the default frontstage). Wait for it to be loaded into the list before rendering the content
```

Comment:
should probably remove the env var from  itwin-viewer/packages/apps/viewer-sample-react/.env

Reply:
Nice catch. Didn't realize we were using that. Removed

Reply:
do you need to do a rush change for the app pkg?

Reply:
its not necessary. Its ignored since the pkg isn't published (still 1.0.0)